docs/admin/kubeadm-token.md

```diff
@@ -0,0 +1,46 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm token
+notitle: true
+---
+{% capture overview %}
+# Kubeadm token
+Bootstrap Tokens are used for establishing bidirectional trust between a node joining 
+the cluster and a the master node, as described in [Authenticating with Bootstrap Tokens](bootstrap-tokens.md).
+
+`kubeadm-init` creates an initial token with 24h TTL; following commands allow to manage 
```

Comment:
.. .24h TTL. The following commands allow you to manage such a token and also to create and manage new ones.